dynamiq/nodes/agents/base.py

```diff
@@ -254,15 +257,13 @@ def _run_llm(self, prompt: str, config: RunnableConfig | None = None, **kwargs)
                 **kwargs,
             )
             self._run_depends = [NodeDependency(node=self.llm).to_dict()]
-            logger.debug(f"Agent {self.name} - {self.id}: RAW LLM result:\n{llm_result.output['content']}")
             if llm_result.status != RunnableStatus.SUCCESS:
                 raise ValueError("LLM execution failed")
```

Comment:
Provide information about error in raised exception. `raise ValueError("LLM execution failed: {llm_result.output['content']}})`